### ISSUE TITLE
Nudged/Rounding Issue

### ISSUE THREAD

**Issue by glitchassassin** [flauwekeul/honeycomb#43]
This comes up in calculating hex lines. Sample code to demonstrate the issue:

```
let original = Honeycomb.extendHex()(-3, -1)
let nudged = original.nudge()
let rounded = nudged.round()

document.write(`
Original: ${original.q}, ${original.r}, ${original.s} <br/>
Nudged: ${nudged.q}, ${nudged.r}, ${nudged.s} <br/>
Rounded: ${rounded.q}, ${rounded.r}, ${rounded.s}
`);
```

Expected behavior: Original and Nudged should round to the same value
Actual behavior:

```
Original: -2, -1, 3
Nudged: -2.999999, -0.999999, 3.9999979999999997
Rounded: -3, -1, 4
```

**Comment by glitchassassin** [comment]
Added PR #44 for this issue.

**Comment by flauwekeul** [comment]
Thanks for finding the bug and opening the issue and PR. I'll look into it soon.

**Comment by flauwekeul** [comment]
There was indeed a bug in the `nudge()` method, but I solved it a little differently than your PR #44. Thanks for pointing it out! I'll release a patch with the fix soon.
